pages/send/ConfirmPayment.tsx

```diff
@@ -45,6 +54,13 @@ export function ConfirmPayment() {
       const response = await nwcClient.payInvoice({
         invoice,
         amount: amount ? amountToPaySats * 1000 : undefined,
+        metadata: {
+          recipient_data: {
+            identifier: recipientIdentifier,
+            description: recipientDescription,
```

Comment:
I think the naming and where description is set here is not great. It comes from the LUD-06 metadata for the invoice. I don't really think this should be in `recipient_data`?

Reply:
Maybe just remove this from the PR.